File: JavaSolutions/levelOne/Mississippi.java
package semicolonClassWork.levelOne;

public class Mississippi {
    public static void main(String[] args) {

        String word = "Mississippi";
        int iCount = 0;
        int sCount = 0;
        for (int index = 0; index < word.length(); index++){
          char letter = word.charAt(index);
          if (letter == 'i'){
              iCount++;
          }
          if (letter == 's'){
              sCount ++;
          }
        }
        System.out.printf("i occur %s times%n", iCount);
        System.out.printf("s occur %s times", sCount);
    }
}

File: JavaScript Solution/level one/mississippi.js
var word = "Mississippi";
let iCount = 0;
let sCount = 0;
for (var index = 0; index < word.length; index++){
    if (word.charAt(index) == 'i') {
        iCount++;
    }
    if (word.charAt(index) == 's') {
        sCount++;
    }
}

console.log("Letter i occur " + iCount)
console.log("Letter s occur " +sCount)